src/panels/ActionsPanel/ActionsPanel.tsx

```diff
@@ -72,6 +71,8 @@ type ActionsQueryParams = {
   units: string[];
 };
 
+type ConfirmTypes = "submit" | null;
```

Comment:
I was thinking that maybe we could create a `types.ts` file in `panels` folder and move this one there, as this exact type is utilized in `CharmActionsPanel`. Also, we utilize `type ConfirmTypes = "apply" | "cancel" | null` in `ConfigPanel.tsx` and I was thinking if maybe it might make sense to make a common one for all of them that looks something like `type ConfirmTypes = "submit" | "cancel" | null`?

Reply:
I think that would be fine to do.